### PR TITLE
Upgraded Horion to VS 2022 and changed some modules to onLevelRender which makes them better and faster

### PR DESCRIPTION
**Pull request by Intoprelised**
onLevelRender is better and faster

### CODE REVIEW

**Comment by Intoprelised** [comment]
what is the error

**Comment by Intoprelised** [comment]
ohhh the upgrading

**Comment by N0tExisting** [comment]
> ohhh the upgrading

Update the Workflow I guess

**Comment by Intoprelised** [comment]
> > ohhh the upgrading
> 
> Update the Workflow I guess

yeah

**Comment by riesters** [comment]
Horion is dead, no one is going to accept your pull request.

**Comment by Intoprelised** [comment]
> Horion is dead, no one is going to accept your pull request.

😎😳

File: Horion/Module/Modules/CrystalAura.cpp [hunk 1]
```diff
@@ -113,7 +113,7 @@ void CrystalAura::DestroyC(C_Entity* ent, int range) {
 }
 
 bool shouldChange = false;
-void CrystalAura::onTick(C_GameMode* gm) {
```
Comment (N0tExisting) [inline Horion/Module/Modules/CrystalAura.cpp:116]: No, you might place more than one crystal at once

**Review by N0tExisting — CHANGES_REQUESTED** [review]
Idk, why you are using `onLeveRender` instead of `onTick`, you really shouldn't, it might break stuff horribly.

You also need to change the build workflow

File: Horion/Module/Modules/CrystalAura.cpp (hunk 1 above)
Comment (Intoprelised) [reply Horion/Module/Modules/CrystalAura.cpp:116]: Thansk for the feedback il re make this a different time and better

**Pull request closed without merging**